content/docs/alerting/alertmanager.md

```diff
@@ -67,3 +67,14 @@ Silences are configured in the web interface of the Alertmanager.
 The Alertmanager has [special requirements](../clients) for behavior of its
 client. Those are only relevant for advanced use cases where Prometheus
 is not used to send alerts.
+
+## High Availability
+
+Alertmanager supports a mesh configuration to create a cluster for high availability.
+This can be configured using the [-mesh-*](https://github.com/prometheus/alertmanager#high-availability) flags.
+
+It's important not to load balance traffic between Prometheus and its Alertmanagers, but instead, point Prometheus to a list of all Alertmanagers.
+
+The Alertmanager implementation expects all alerts to be sent to all Alertmanagers to ensure high availability.
+You can use the `-alertmanager.url` when starting
```

Comment:
This is outdated with Prometheus 2.0 coming along which removed this flag. This also belongs with docs on how to hook the two together, not just in the HA section. I'd remove this.